ios/FluentUI/PersonaButton/PersonaButtonTokenSet.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Pre-defined sizes of the persona button
+@objc public enum MSFPersonaButtonSize: Int, CaseIterable {
+    case small
+    case large
+
+    var avatarSize: MSFAvatarSize {
+        switch self {
+        case .large:
+            return .xxlarge
+        case .small:
+            return .xlarge
+        }
+    }
+
+    var shouldShowSubtitle: Bool {
+        switch self {
+        case .large:
+            return true
+        case .small:
+            return false
+        }
+    }
+}
+
+/// Representation of design tokens to persona buttons at runtime which interfaces with the Design Token System auto-generated code.
```

Comment:
Should the "persona button"s be `PersonaButton`s?

Reply:
Also, not sure if it was intended to be a comment about the future, but does it really interface with the "Design Token System auto-generated code" at the moment?